app/_data/docs_nav_konnect.yml

```diff
@@ -171,12 +171,12 @@
   items:
     - text: Introduction to Analytics
       url: /analytics/
-    - text: Summary Dashboard
-      url: /analytics/summary-dashboard/
     - text: Analyze Services and Routes
       url: /analytics/services-and-routes/
-    - text: Generate Reports
-      url: /analytics/generate-reports
+    - text: Reports Use Cases
+      url: /analytics/use-cases
+    - text: Reference
```

Comment:
Can we combine "User Cases" and "Reference" under a "Reports" menu?

Reply:
@sichvoge We had something similar to that when we first started, but decided against going with that because:

- This Analytics section is already pretty small, so subsections wouldn't be necessary to break up the content (and subsections would kind of "hide" the docs from users)
- It was easy enough to put "Reports" in the page titles, so we felt that it clued the reader in to what the page was about without needing a separate subsection. 

I can add "Reports" in front of the "Reference" page title in the nav so that's clearer about what that page is a reference for.

Reply:
OK. Yes, if you can at least add "Reports" in front of the "Reference" page title - that would be awesome.